conduct.md

```diff
@@ -24,15 +21,14 @@ As a member of the community, you agree to the following:
 
 **Encouraged:**
 
-- Be kind and courteous.
-- Respect differences of opinion and remember that every design or implementation choice carries a trade-off and numerous costs. There is seldom a single right answer.
-- Remember that everyone was new to Scala at some point. We want to encourage newcomers to join our community and learn the Scala language and ecosystem. Assume competence.
-- Show empathy towards other community members.
+- **Be kind and courteous.** Keep in mind that all public communication is received by many people you don’t know, so before sending a message please ask yourself whether someone from a different context would misunderstand it. Do not attack other persons, in particular consider that age, body size, disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, or religion have no bearing on the respect you owe another person.
+- **Respect differences of opinion** and remember that every design or implementation choice carries a trade-off and numerous costs. There is seldom a single right answer, we will find the best solutions by engaging in constructive discussion, with everybody bringing their unique viewpoint and experience to the table.
+- **Remember that everyone was new to Scala at some point.** We want to encourage newcomers to join our community and learn the Scala language and ecosystem. Always assume good intentions and a willingness to learn, just as you are willing to evolve your own opinion as you gain new insights.
 
 **Discouraged:**
 
 - Keep unstructured critique to a minimum. We encourage sharing ideas and perspectives, so please ensure that your feedback is constructive and relevant. If you have solid ideas you want to experiment with, make a fork and see how it works.
-- Avoid aggressive and micro-aggressive behavior, such as unconstructive criticism, providing corrections that do not improve the conversation (sometimes referred to as "well actually"s), repeatedly interrupting or talking over someone else, feigning surprise at someone’s lack of knowledge or awareness about a topic, or subtle prejudice (for example, comments like “That’s so easy my grandmother could do it.”). For more examples of this kind of behavior, [see the Recurse Center's user manual](https://www.recurse.com/manual#sec-environment).
+- Avoid aggressive behavior, such as unconstructive criticism, providing corrections that do not improve the conversation (sometimes referred to as "well actually"s), repeatedly interrupting or talking over someone else, feigning surprise at someone’s lack of knowledge or awareness about a topic, or subtle prejudice (for example, comments like “That’s so easy my grandmother could do it.”). For more examples of this kind of behavior, [see the Recurse Center's user manual](https://www.recurse.com/manual#sec-environment).
```

Comment:
I don't have a strong feeling about the word "micro-aggressive" either way, but I would like to know what the other original drafters of the CoC (@heathermiller, @robinske, @stewSquared — apologies if I'm forgetting anybody, it's been a while) think.

"Micro-aggressive" as a term is a rather recent invention and is arguably jargon. In that sense, I wouldn't mind losing it. But before losing it, I think we should consider whether the word was doing some work in the sentence that we ought to try and accomplish in some other way. Or maybe the rest of the paragraph already does that work? I'm not sure.

The reason the term "micro-aggressive" even exists is because of a dynamic which I believe is important and real. Namely, blatant open aggression isn't the only way an environment can be hostile. It can also be hostile if more subtle forms of aggression are repeated often enough over time. By simply deleting "and micro-aggressive", that shade of meaning is lost.